evennia/commands/command.py

```diff
@@ -423,3 +427,22 @@ def get_extra_info(self, caller, **kwargs):
         if hasattr(self, 'obj') and self.obj != caller:
             return " (%s)" % self.obj.get_display_name(caller).strip()
         return ""
+
+    def get_help(self, caller, cmdset):
+        """
+        Return the help message for this command and this caller.
+
+        By default, return self.__doc__ (the docstring just under
+        the class definition).  You can override this behavior,
+        though, and even customize it depending on the caller.
+
+        Args:
+            self: the command itself.
```

Comment:
See the Evennia style guide [here](https://github.com/evennia/evennia/blob/master/CODING_STYLE.md#function--method-docstrings). 
 - `self` should not be included as an argument
 - All arguments should list its type within parentheses. it comes to Evennia entities, I usually use `Object` rather than `ObjectDB` for clarity, so for example `caller (Object or Player)`. The `cmdset` should have `CmdSet` as type. 
 - Since the return is not implicit, the `Returns` block should be listed, something like 
    ```
    Returns:
        docstring (str): The help text to return for this command. 
    ```